src/GitTfs.Vs2022/GitTfs.Vs2022.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\CommonAssemblyInfo.cs">
+      <Link>Properties\CommonAssemblyInfo.cs</Link>
+    </Compile>
+    <Compile Include="..\GitTfs.VsCommon\TfsPlugin.cs">
+      <Link>TfsPlugin.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\GitTfs\GitTfs.csproj" />
+	<ProjectReference Include="..\GitTfs.Vs2019\GitTfs.Vs2019.csproj" />
```

Comment:
This referebce us wrong. 
The GitTFS.VsXXXX assemblies don't reference any of the other GitTFS.VsXXXX assemblies, as these assemblies are 
there to isolate the different VS assemblies which are comming from the different Visual Studio versions.

I assume that with that reference, you are actually using the TFS access libararies from VS2019, not the ones from VS2022

Reply:
Removed this reference. Fixed dependencies.